src/runtime/qurt_hvx.cpp

```diff
@@ -2,6 +2,7 @@
 #include "HalideRuntimeQurt.h"
 #include "printer.h"
 #include "mini_qurt.h"
+#include "hexagon_remote/log.h"
```

Comment:
I'm just surprised that the remote runtime is able to resolve the log_printf symbol. At the very least, on the simulator, dlsym doesn't do anything, so without a 'log_printf' symbol in this table: https://github.com/halide/Halide/blob/master/src/runtime/hexagon_remote/sim_remote.cpp#L142 I don't see how it can resolve it... either way, definitely use debug(user_context) when writing debug info from the runtime. It's definitely not designed to work, and so it might break if we try to improve the runtime in the future.